Mixmatch/.ipynb_checkpoints/README-checkpoint.md

```diff
@@ -0,0 +1,143 @@
+# MixMatch - A Holistic Approach to Semi-Supervised Learning
+
+Code for the paper: "[MixMatch - A Holistic Approach to Semi-Supervised Learning](https://arxiv.org/abs/1905.02249)" by David Berthelot, Nicholas Carlini, Ian Goodfellow, Nicolas Papernot, Avital Oliver and Colin Raffel.
+
+This is not an officially supported Google product.
+
+## Setup
```

Comment:
Is this setup from MixMatch code repo or do you make it yourself? If it's the former case, please add a comment to address it.

Reply:
First of all, this is also a checkpoint generated by jupyterlab. And yes, this line is originated from the Mixmatch code repo, I will make a change in the readme file.

Reply:
Yes please clarify which part is from the original MixMatch instruction and which part is from you in this readme file.